lock.sbt

```diff
@@ -29,26 +29,25 @@ dependencyOverrides in ThisBuild ++= Set(
   "com.github.jnr" % "jnr-x86asm" % "1.0.2",
   "com.github.swagger-akka-http" % "swagger-akka-http_2.12" % "0.9.1",
   "com.google.code.findbugs" % "annotations" % "2.0.1",
-  "com.google.guava" % "guava" % "19.0",
+  "com.google.guava" % "guava" % "20.0",
   "com.h2database" % "h2-mvstore" % "1.4.194",
   "com.iheart" % "ficus_2.12" % "1.4.0",
-  "com.ning" % "async-http-client" % "1.9.11",
   "com.spotify" % "docker-client" % "8.1.2",
   "com.thoughtworks.paranamer" % "paranamer" % "2.8",
   "com.typesafe" % "config" % "1.3.1",
   "com.typesafe" % "ssl-config-core_2.12" % "0.2.1",
-  "com.typesafe.akka" % "akka-actor_2.12" % "2.4.16",
-  "com.typesafe.akka" % "akka-http-core_2.12" % "10.0.1",
-  "com.typesafe.akka" % "akka-http-testkit_2.12" % "10.0.3",
-  "com.typesafe.akka" % "akka-http_2.12" % "10.0.1",
-  "com.typesafe.akka" % "akka-parsing_2.12" % "10.0.1",
+  "com.typesafe.akka" % "akka-actor_2.12" % "2.4.17",
+  "com.typesafe.akka" % "akka-http-core_2.12" % "10.0.5",
+  "com.typesafe.akka" % "akka-http-testkit_2.12" % "10.0.5",
+  "com.typesafe.akka" % "akka-http_2.12" % "10.0.5",
+  "com.typesafe.akka" % "akka-parsing_2.12" % "10.0.5",
   "com.typesafe.akka" % "akka-persistence-query-experimental_2.12" % "2.4.16",
   "com.typesafe.akka" % "akka-persistence_2.12" % "2.4.16",
```

Comment:
There are still two different (supposedly compatible though) versions of akka jars in the classpath :(